data/json/mapgen/veterinarian.json

```diff
@@ -0,0 +1,203 @@
+[
+{
+        "type": "vehicle_group",
+        "id": "VETS",
+        "vehicles": [
+            [ "car", 300 ],
+            [ "pickup", 100 ],
+			[ "suv", 50 ]
+        ]
+    },{
+        "type": "item_group",
+        "id": "vet_softdrug",
+        "items": [
+            ["bandages", 50],
+            ["1st_aid", 10],
+			["saline", 20],
+			["gummy_vitamins", 15],
+			["nyquil", 10],
+			["disinfectant", 15],
+			["vaccine_shot", 10],
+			["protein_powder", 30],
+			["smoxygen_tank", 20],
+			["elec_hairtrimmer", 10],
+			["survnote", 5],
+			["mask_dust", 65],
+			["glasses_safety", 10],
+			["towel", 50],
+			["scissors", 50],
+			["fungicide", 10],
+			["soap", 20],
+			["chem_hydrogen_peroxide", 10],
+			["canister_empty", 5],
+			["stethoscope", 5],
+			["scalpel", 55],
+			["gloves_medical", 50],
+			["medical_gauze", 25],
+			["quikclot", 10],
+			["hacksaw", 5],
+			["gloves_rubber", 25],
+			["knife_butcher", 5],
+			["machete", 5],
```

Comment:
With such a low chance of spawning, they shouldn't be regular events.  I justified their presence the same why I justified the Sienfeld reference or how pizzerias justify having 9mm handguns in the trashcan